packages/react-components/react-progress/src/components/Progress/useProgress.tsx

```diff
@@ -13,56 +13,29 @@ import type { ProgressProps, ProgressState } from './Progress.types';
  */
 export const useProgress_unstable = (props: ProgressProps, ref: React.Ref<HTMLElement>): ProgressState => {
   // Props
-  const { thickness = 'medium', indeterminate = false, percentComplete = 0 } = props;
-  const baseId = useId('progress-');
+  const { thickness = 'medium', value, max = 1.0 } = props;
 
   const root = getNativeElementProps('div', { ref, role: 'progressbar', ...props });
 
-  const label = resolveShorthand(props.label, {
-    defaultProps: {
-      id: baseId + '__label',
-    },
-  });
-
-  const description = resolveShorthand(props.description, {
-    defaultProps: {
-      id: baseId + '__description',
-    },
-  });
-
   const bar = resolveShorthand(props.bar, {
     required: true,
     defaultProps: {
-      'aria-valuemin': indeterminate ? undefined : 0,
-      'aria-valuemax': indeterminate ? undefined : 100,
-      'aria-valuenow': indeterminate ? undefined : Math.floor(percentComplete),
+      'aria-valuemin': value ? 0 : undefined,
+      'aria-valuemax': value ? max * 100 : undefined,
+      'aria-valuenow': value ? Math.floor(value!! * 100) : undefined,
```

Comment:
Since `max` can be different than `1`, should you be calculating the percentages based on the max value and not assume `1`?